src/tools/auth0/handlers/actions.ts

```diff
@@ -131,10 +132,10 @@ export default class ActionHandler extends DefaultAPIHandler {
           log.info(`[${this.type}]: Waiting for build to complete ${this.objString(action)}`);
           action.retry_count = 1;
         }
-        if (action.retry_count > MAX_ACTION_DEPLOY_RETRY) {
+        if (action.retry_count > MAX_ACTION_DEPLOY_RETRY_ATTEMPTS) {
           throw err;
         }
-        await sleep(1000);
+        await sleep(2000);
```

Comment:
From my limited testing, I found that an action either built very quickly, almost immediately or took a long time, I believe retrying every second is unnecessary.